TranslatorReasonerAPI.yaml

```diff
@@ -784,10 +796,69 @@ components:
           items:
             $ref: '#/components/schemas/Attribute'
           nullable: true
+        qualifiers:
+          type: array
+          description: >-
+            A list of qualifiers that together provide additional detailed
+            nuance to the assertion beyond what the subject-predicate-object
+            can provide.
+          items:
+            $ref: '#/components/schemas/Qualifier'
+          nullable: true
       additionalProperties: false
       required:
         - subject
         - object
+    Qualifier:
+      additionalProperties: false
+      description: >-
+        An additional nuance attached to an assertion
+      type: object
+      properties:
+        qualifier_type_id:
+          type: string
+          description: >-
+            The category of the qualifier, drawn from a hierarchy of qualifier
+            slots in the Biolink model (e.g. subject_aspect, subject_direction,
+            object_aspect, object_direction, etc).
+          example: subject_aspect
+          nullable: false
+        value:
+          type: string
+          description: >-
+            The value associated with the type of the qualifier, drawn from
+            a set of controlled values by the type as specified in
+            the Biolink model (e.g. 'decreased' or 'increased' for the type
+            subject_direction, etc).
+          example: decreased
+          nullable: false
+      required:
+        - type
+        - value
+    QualifierSet:
+      additionalProperties: false
+      description: >-
+        A set of Qualifiers that act together to add nuance to an assertion.
```

Comment:
With the renaming/reframing of this object, update definition to something like: 
"Defines a query constraint based on the type and value of a Qualifier attached to an edge.   e.g. can constrain a "ChemicalX - affects ?Gene" query to return only edges where ChemicalX affects the *expression* of some Gene, by specifying a qualifier_type of "object_aspect", and value of "expression".

Reply:
Qualifier is used both for QEdges and Edges. So it used both for querying and for conveying assertions. That is why the definition is slanted more toward the assertion side. In the context of queries, it specifies the constraints on the desired assertions. So okay as is?

Reply:
True that 'Qualifier' object is used in asserting knowledge and querying for it.  But the definition here is for a 'QualifierSet' object, which we agreed above to rename 'QualifierConstraint' (because its express purpose is to allow constraining queries based on the values of Qualifiers on an edge).  
So given that we are not defining a  'QualifierConstraint' object, it makes sense to me to update the definition so that it clearly conveys this purpose of this object in constraining queries.

Reply:
I understand now. Yes, agreed. Updated with minor edits. See if the committed version seems agreeable.